modules/core/04-channel/keeper/upgrade.go

```diff
@@ -153,9 +158,10 @@ func (k Keeper) ChanUpgradeTry(
 	); err != nil {
 		return types.Upgrade{}, errorsmod.Wrap(err, "failed to verify counterparty channel state")
 	}
-
-	if err := k.syncUpgradeSequence(ctx, portID, channelID, channel, counterpartyUpgradeSequence); err != nil {
```

Comment:
is this relevant now? It appears this was last usage (and looking through spec I can't see any `!=` checks for upgrade sequences.